SportsHub/SportsHub.Api/Validations/CommentValidator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FluentValidation;
+using SportsHub.AppService.Authentication.Models.DTOs;
+using SportsHub.DAL.Data.Configurations.Constants;
+using SportsHub.Domain.Constants;
+
+namespace SportsHub.Api.Validations
+{
+    public class CommentValidator : AbstractValidator<CreateCommentDTO>
+    {
+        public CommentValidator()
+        {
+            RuleFor(x => x.Content)
+                .NotEmpty().WithMessage(ValidationMessages.CommentContentValidationNotEmptyMessage)
+                .Length(0, ConfigurationConstants.CommentContentMaxLength)
+                .WithMessage(ValidationMessages.CommentContentValidationLengthMessage.Replace("[0]", ConfigurationConstants.CommentContentMaxLength.ToString()));
```

Comment:
Please take a look as string.Format(). It is better to use it and not Replace.

Reply:
Updated.